atomic_reactor/plugins/exit_sendmail.py

```diff
@@ -142,6 +152,31 @@ def __init__(self, tasker, workflow,
         else:
             self.log.info("Koji connection established")
 
+    def _fetch_full_logs(self):
+        if not self.osbs:
+            namespace = get_build_json().get('metadata', {}).get('namespace', None)
+            osbs_conf = Configuration(conf_file=None, openshift_uri=self.url,
+                                      use_auth=self.use_auth, verify_ssl=self.verify_ssl,
+                                      namespace=namespace)
+            self.osbs = OSBS(osbs_conf, osbs_conf)
+            res = self.workflow.prebuild_results.get(PLUGIN_KOJI_PARENT_KEY) or {}
+            build_info = res.get('parent-image-koji-build') or {}
+            self.build_id = build_info.get('id')
```

Comment:
`build_id` doesn't seem to be used outside of this method. Make it just `build_id` instead of `self.build_id`?

Reply:
Fixed.